src/libs/OptionsListUtils.js

```diff
@@ -449,6 +450,8 @@ function createOption(logins, personalDetails, report, reportActions = {}, {show
 
         if (result.isChatRoom || result.isPolicyExpenseChat) {
             result.alternateText = showChatPreviewLine && !forcePolicyNamePreview && lastMessageText ? lastMessageText : subtitle;
+        } else if (result.isMoneyRequestReport) {
+            result.alternateText = Localize.translate(preferredLocale, 'footer.expenseReports');
```

Comment:
I think we need "Expense report" here (or anything related that is not coming from `footer`)

Reply:
Yeah, I think we need a new key for that. Ask on Slack for translation if needed.

Reply:
I think we need to change this further as IOU is not expense report

cc @trjExpensify what should we use for the IOU? `Money request`?

Reply:
The preview message (i.e the second line in the search results) for an expense / iouReports should be the latest comment added to the report. The same as what would show in the LHN. I don't think there's a scenario where we would ever show `Expense report` or `Money request`.